dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -766,6 +817,14 @@ public static void addRollbackListener(DotRunnable listener) throws DotHibernate
 		}
     }
 
+	public static void closeSessionSilently() {
+
+		try {
+			closeSession();
+		} catch (DotHibernateException e) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)